File: app/src/main/java/com/hey/mietunoff/mietunofficial/model/schedule/Time.java
package com.hey.mietunoff.mietunofficial.model.schedule;

/**
 * Contains information about schedule time exercise
 * Part of {@link Data}
 */
public class Time {

    /**
     * String of representation of number of exercise in all schedule time
     */
    private String Time;

    /**
     * Lesson time start
     */
    private String TimeFrom;

    /**
     * Lesson time end
     */
    private String TimeTo;

    /**
     * number of exercise in all schedule time
     */
    private String Code;

    public String getTime() {
        return Time;
    }

    public void setTime(String Time) {
        this.Time = Time;
    }

    public String getTimeFrom() {
        return TimeFrom;
    }

    public void setTimeFrom(String TimeFrom) {
        this.TimeFrom = TimeFrom;
    }

    public String getTimeTo() {
        return TimeTo;
    }

    public void setTimeTo(String TimeTo) {
        this.TimeTo = TimeTo;
    }

    public String getCode() {
        return Code;
    }

    public void setCode(String Code) {
        this.Code = Code;
    }
}

File: miet-api/src/main/java/com/alex/miet/miet_api/model/Time.kt
package com.alex.miet.miet_api.model


import com.google.gson.annotations.SerializedName

data class Time(

    /**
     * number of exercise in all schedule time
     */
    @SerializedName("Code") val code: Int,
    /**
     * String of representation of number of exercise in all schedule time
     */
    @SerializedName("Time") val time: String,
    /**
     * Lesson time start
     */
    @SerializedName("TimeFrom") val timeFrom: String,
    /**
     * Lesson time end
     */
    @SerializedName("TimeTo") val timeTo: String
)